src/app/productConfig/productConfigForm/productConfigForm.component.js

```diff
@@ -340,6 +340,12 @@ class ProductConfigFormController {
       if (includes(error.data, 'already in the cart')) {
         this.errorAlreadyInCart = true
         this.onStateChange({ state: 'errorAlreadyInCart' })
+      } else if (error.data && error.data.messages && error.data.messages[0] && error.data.messages[0].id === 'field.invalid.decimal.format') {
+        this.amountFormatError = error.data.messages[0]['debug-message']
```

Comment:
Do you need to restart the `this.amountFormatError = ''` at the start of the `saveGiftToCart()`?